public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Don't need this too as of now

Reply:
You can remove the React logos. The AnitaB.org logo should work for us

Reply:
should I delete the manifest.json file or just its content related to react logo?

Reply:
added the AnitaB icon instead for tab icon to replace react favicon icon instead 😂 
Hope this is Ok, @meenakshi-dhanani 
<img width="1318" alt="Screen Shot 2020-06-03 at 9 35 05 pm" src="https://user-images.githubusercontent.com/29667122/83632437-9e955480-a5e2-11ea-931c-944ec7edf85a.png">